src/content/en/fundamentals/performance/rail.md

```diff
@@ -145,16 +146,21 @@ To evaluate your site against RAIL metrics, use the Chrome DevTools [Timeline to
       <td data-th="User Test">User taps a button (for example, opening navigation).</td>
     </tr>
     <tr>
-      <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 100ms for initial response.</td>
-      <td data-th="User Test">User initiates page scroll or animation begins.</td>
-    </tr>
-    <tr>
       <td data-th="RAIL Step"><strong>Animation</strong></td>
       <td data-th="Key Metric">Each frame's work (from JS to paint) completes < 16ms.</td>
       <td data-th="User Test">User scrolls the page or sees an animation.</td>
     </tr>
     <tr>
+      <td data-th="RAIL Step"><strong>Animation</strong></td>
+      <td data-th="Key Metric">Input latency (from tap to paint) < 16ms.</td>
```

Comment:
"from tap to paint" is incorrect. This should refer to scroll or drag, probably.

Or perhaps we can just merge the two Animation rows?